frontend/src/pages/detail/components/ReviewItem/ReviewItem.tsx

```diff
@@ -0,0 +1,79 @@
+import styled from '@emotion/styled';
+import filledStar from '../../../../common/assets/images/starIcon.svg';
+import emptyStar from '../../../../common/assets/images/emptyStarIcon.svg';
+import { ReviewItemType } from '../../types/ReviewResponse';
+
+function ReviewItem({ review }: { review: ReviewItemType }) {
+  const { reviewerName, createdAt, rating, content } = review;
+  const formattedDate = createdAt.split('-');
```

Comment:
p4: 이 방법도 간편하고 좋은 거 같아 👍👍

저번에 만들었던 포맷팅 방법은 서버에서 주는 date 데이터에 -를 쓰지 않을 때를 고려했던 거 같아! 

지금처럼 배열로 쓴다면 인덱스로 접근 하기 보다는 배열로 바로 값을 꺼내서 쓰는 것도 괜찮을 거 같아 👍


```suggestion
  const [year, month, day] = createdAt.split('-');
```

Reply:
아하 그런 부분까지 고려하고 코드를 짜는구나... 많이 배워갑니다..
배열로 값을 꺼내서 쓰는것도 너무 좋다!!! 땡큐!!!!